tests/api-watch/test_fully_expand_type_expr.ml

```diff
@@ -39,28 +39,34 @@ let%expect_test "test_fully_expand_type_expr_on_alias_types" =
   let v1_type = String_map.find "v1" value_map in
   let path, args = get_tconstr v1_type in
   let expanded_type_expr =
-    Typing_env.fully_expand_type_expr ~typing_env:env ~type_expr:v1_type ~path
-      ~args
+    Typing_env.fully_expand_tconstr ~typing_env:env ~path ~args
   in
-  Printtyp.type_expr Format.std_formatter expanded_type_expr;
-  [%expect {| (string * int) list |}]
+  match expanded_type_expr with
+  | None -> assert false
+  | Some expr ->
+      Printtyp.type_expr Format.std_formatter expr;
+      [%expect {| (string * int) list |}]
 
 let%expect_test "test_fully_expand_type_expr_on_nominal_types" =
   let v2_type = String_map.find "v2" value_map in
   let path, args = get_tconstr v2_type in
   let expanded_type_expr =
-    Typing_env.fully_expand_type_expr ~typing_env:env ~type_expr:v2_type ~path
-      ~args
+    Typing_env.fully_expand_tconstr ~typing_env:env ~path ~args
```

Comment:
Sorry, that's not what I meant. We should keep the existing single step expansion tests and add an extra one that tests multiple levels of expansion.

Reply:
I already test both. [Here](https://github.com/ocaml-semver/ocaml-api-watch/pull/148/files#diff-9b8adf3218f9ac231d6ff38ed29f2aa45cf21b5438ef7873023dd0370eb7680c) I test the single step expansion, and [here](https://github.com/ocaml-semver/ocaml-api-watch/pull/148/files#diff-2045d9faff87dc0cbd188de3f528977c93fe773efe8a19cfe7ede9d668ad7f97) I test the full expansion.
Could you provide an example with a case that you want to test ?

Reply:
my bad, the diff was shown as just a rename!